packages/shader-compiler/src/bundler/index.ts

```
@@ -0,0 +1,91 @@
+
+  return {
+    name: "shader-compiler",
+
+    async buildStart() {
+      if (!precompileOptions || initialDone) return;
+      initialDone = true;
+      await runFull(precompileOptions);
+
+      if (this.meta.watchMode && !watcherStarted) {
+        watcherStarted = true;
+        startWatcher(precompileOptions).catch((e) => this.warn(`[shader-compiler] watcher failed: ${(e as Error).message}`));
```

ESLint

[error] Replace `·this.warn(`[shader-compiler]·watcher·failed:·${(e·as·Error).message}`)` with `⏎··········this.warn(`[shader-compiler]·watcher·failed:·${(e·as·Error).message}`)⏎········`

(prettier/prettier)